### ISSUE TITLE
Need to escape or deny ":" in usernames, repos, etc.

### ISSUE THREAD

**Issue by Southern** [spacedock-io/docker-index#28]
If we're going to be using ":" in the database documents, we need to either escape or deny it in usernames, repositories, etc.

This doesn't necessarily have to be done before the MVP, which is why I haven't assigned this to the MVP milestone. We absolutely need to have this fixed by production.

**Comment by AvianFlu** [comment]
No longer relevant, postgres prevents us from needing such things
